scaper/core.py

```diff
@@ -1355,6 +1416,7 @@ def _instantiate_event(self, event, isbackground=False,
             allowed_labels = self.fg_labels
 
         # determine label
+        # special case: choose tuple with empty list
         if event.label[0] == "choose" and not event.label[1]:
```

Comment:
Why isn't this also happening for `choose_weighted` here?

Reply:
`choose` has a unique case when you submit `("choose", [])`, where we need to populate the list from files (or folders) on disk, i.e. we need to modify the distribution tuple before sampling from it.

In all other cases, including `choose_weighted`, there's no need to change the tuple before sampling.